taiga_halo2/src/shielded_ptx.rs

```diff
@@ -66,12 +69,48 @@ struct ShieldedPartialTransactionProxy {
     inputs: Vec<NoteVPVerifyingInfoSet>,
     outputs: Vec<NoteVPVerifyingInfoSet>,
     binding_sig_r: pallas::Scalar,
+    hints: Vec<u8>,
 }
 
 impl ShieldedPartialTransaction {
+    #[cfg(feature = "borsh")]
```

Comment:
It feels like this might be ok to have even if borsh feature is not enabled

Reply:
I won't compile. `TrivialValidityPredicateCircuit` uses the borsh to generate the `ValidityPredicateByteCode`.
Should we set a default serialization feature(serde or borsh) inside Taiga? Adding the feature flag everywhere is annoying if I want to use a serialization underlying.